src/sql/sqlops.proposed.d.ts

```diff
@@ -375,6 +375,16 @@ declare module 'sqlops' {
 				 */
 				content: string | DialogTab[],
 
+				/**
+				 * Register model view content for the dialog
+				 */
+				registerContent(handler: (view: ModelView) => void): void;
```

Comment:
What's the expected behavior if the user makes multiple calls to `registerContent`?

Reply:
I added logic to not register if already registered 

---
In reply to: [188375177](https://github.com/Microsoft/sqlopsstudio/pull/1415#discussion_r188375177) [](ancestors = 188375177)